sdk/python/using-mlflow/deploy/mlflow_sdk_online_endpoints_progresive.ipynb

```diff
@@ -0,0 +1,794 @@
+{
+ "cells": [
+  {
```

Comment:
Overall this an amazing notebook. One thing I noticed is that notebook mentions to use AML SDK to get endpoint auth key. Otherwise rest of the notebook doesn't depend on AML SDK. Is there a way this can be achieved using mlflow sdk ? or by using mlflow sdk for invoking the endpoint rather than curl ?

Reply:
Indeed. I couldn't find a way to retrieve the authentication key. I don't know if it makes sense to return them as part of the endpoint properties?

Reply:
deployment_client.predict should take care of it internally depending the auth used for enpoint. Would that be an option?

Reply:
Added!